### PR TITLE
GANGS

### PR DESCRIPTION
**Pull request by Manly-Dwarf**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Kawaii-Big-Boss
add: Re-added Pre-Rebase Gangs!
add: Also added a pirate gang. Yarr!
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

# DO NOT MERGE THIS, THIS IS NOT EVEN KIND OF DONE YET

Gangs is coming back! This PR is 0% done and I'm simply putting it up so people can watch the progress and suggest changes as I go.

### CODE REVIEW

File: hippiestation.dme [hunk 1]
```diff
@@ -2616,7 +2617,6 @@
 #include "hippiestation\code\game\gamemodes\vampire\vampire_powers.dm"
 #include "hippiestation\code\game\gamemodes\wizard\spellbook.dm"
 #include "hippiestation\code\game\gamemodes\wizard\spellbook_darkness.dm"
-#include "code\modules\antagonists\xeno\xeno.dm"
```
Comment (Manly-Dwarf) [inline hippiestation.dme]: I don't even know what the hell

**Comment by Manly-Dwarf** [comment]
What I'm doing right now is just porting in the code from oldgangs, which will later be cleaned up/made compatible. Don't expect any of this to compile yet!

**Comment by yoyobatty** [comment]
Looks like you've got 2 errors on your travis build:
hippiestation/code/datums/mind.dm:58:error: ticker.mode.remove_gangster: undefined var
hippiestation/code/datums/mind.dm:59:error: remove_objectives: undefined proc

I suggest looking at other game modes to see what procs got redefined and find the old code that got tg-fied.

**Comment by Manly-Dwarf** [comment]
All I'm doing right now is porting the code ya nerd. I'm rewriting it later.

**Comment by yoyobatty** [comment]
Which is exactly what I'm implying you should do, get it to work. 

Edit: And it looks like your travis compiled correctly so it should be good to go. Other than modularizing it, I think it's fine the way it is, Gangs was flawless.

**Comment by Manly-Dwarf** [comment]
nein

The only reason it compiled was because I commented everything out.

**Comment by yoyobatty** [comment]
There needs to be a vote held. Old gangs or revised old gangs.

**Comment by Manly-Dwarf** [comment]
There already was. Dwarves + Gangs won with Revised Old Gangs in second, but Revised Old Gangs was safer so I'll learn how to make gamemodes with this, and then move to the bigger one.

**Comment by ghost** [comment]
Trifornt, it appears that you are getting some errors here.

**Comment by ghost** [comment]
Trifornt, this isn't a full game mode? We cna't merge this yet Trifornt... You should work on it more until it's done.

**Comment by ghost** [comment]
Bro stop taking it badly when we are just trying to tell you the truth -- ITS NOT READY TO MERGE


man up and accept that

**Comment by ghost** [comment]
Hello? HELLO?

**Comment by Manly-Dwarf** [comment]
Dude, I said that it wasn't ready to merge in the OP. It's not ready at all. What are you going on about?

**Comment by ghost** [comment]
Yea so I'm telling you that it's not even near complete. Why can't you face the facts?

**Comment by ghost** [comment]
+1

**Comment by praisenarsie** [comment]
wait kbb is trifornt?

:thinking:

**Comment by Carbonhell** [comment]
as i said on discord, it will be WAY easier if you just rewrite it from scratch.

**Comment by Manly-Dwarf** [comment]
I think I'd prefer to take a look at the old code so I have a better idea of what I'm doing. Totally rewriting it would take a really long time.

**Comment by yoyobatty** [comment]
Spank monkey appeared to have a part schitzo convo with himself then got mad at the first person who replied

**Comment by Manly-Dwarf** [comment]
I apologize for the long delays on this, there's been stuff going on in my life and all.

**Comment by JohnGinnane** [comment]
homey don't rush yourself, this is a big project

**Comment by ghost** [comment]
yah bro what john said it's cool

**Comment by GuyonBroadway** [comment]
@Kawaii-Big-Boss you still working on this shit?

**Comment by Manly-Dwarf** [comment]
Yeah, dude. I'm totally still working on this.

But if you want to take the project over, then by all means, you can. Just remember; the community voted on Pre-Rebase gangs so you have to do that version, not nu-gangs.

**Comment by praisenarsie** [comment]
fixed gonflicts

**Comment by Manly-Dwarf** [comment]
danks good guy praisenarsie but you uncommented shit didn't ye

**Comment by Carbonhell** [comment]
2 months
Have you given up, or are you still working on this?

**Comment by Manly-Dwarf** [comment]
gangs is NEVER happening
jew

**Pull request closed without merging**